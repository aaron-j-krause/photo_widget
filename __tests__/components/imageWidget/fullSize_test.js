import React from 'react'
import renderer from 'react-test-renderer'

import FullSize from '../../../src/components/imageWidget/fullSize'

const testProps = { description: 'test_description', url: 'test_url' }

test('FullSize snapshot', () => {
  const rendered = renderer.create(<FullSize {...testProps} />)

  expect(rendered).toMatchSnapshot()
})
