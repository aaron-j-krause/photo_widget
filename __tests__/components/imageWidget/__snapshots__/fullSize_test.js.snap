exports[`test FullSize snapshot 1`] = `
<img
  alt="test_description"
  className={undefined}
  src="test_url" />
`;
